linera-views/src/memory.rs

```diff
@@ -41,6 +41,15 @@ pub fn create_test_context() -> MemoryContext<()> {
     MemoryContext::new(guard, ())
 }
 
+/// Create a test memory client for working.
```

Comment:
```suggestion
/// Creates a test memory client for working.
```

By "for working" you mean: not just for tests? What's the difference?

Reply:
Ok, corrected.